client/app/components/composites/ListingCard/ListingCard.story.js

```diff
@@ -114,7 +120,7 @@ const ListingCardImageError =
 
 const testPrice = function priceTest(card, mountedCard) {
   it('Should display formatted price', () => {
-    expect(mountedCard.text()).to.include(formatMoney(card.props.listing.price, card.props.listing.priceUnit));
+    expect(mountedCard.text()).to.include(formatMoney(card.props.listing.price.get(':money'), card.props.listing.price.get(':priceUnit')));
```

Comment:
They are harder to write and maintain, if we spread literals everywhere.
But good point, we should have separate tests for utility functions.